nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+
+                                content {
+                                    destination_name = "dgraph-alpha-${alpha_peer.value.id}-grpc-private"
+                                    local_bind_port = alpha_peer.value.grpc_private_port
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-grpc-public"
+                port = "${alpha.value.grpc_public_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-http"
+                port = "${alpha.value.http_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+        }
+    }
+
+    group "grapl-graph-merger" {
+        count = var.graph_mergers
+
+        network {
+            mode = "bridge"
+        }
+
+        task "graph-merger" {
+            driver = "docker"
+
+            config {
+                image = "${var.container_registry}/graph-merger:${var.tag}"
+                args = ["/graph-merger"]
+            }
+
+            env {
+                REDIS_ENDPOINT = "${var.redis_endpoint}"
+                MG_ALPHAS = "${local.alpha_grpc_connect_str}"
+                GRAPL_SCHEMA_TABLE = "${var.schema_table_name}"
+                AWS_REGION = "${var.aws_region}"
+                # https://github.com/grapl-security/grapl/blob/18b229e824fae99fa2d600750dd3b17387611ef4/pulumi/grapl/__main__.py#L165
+                DEST_BUCKET_NAME = "subgraphs-merged-bucket"
+                SOURCE_QUEUE_URL = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-queue"
+                DEAD_LETTER_QUEUE_URL = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-dead-letter-queue"
+            }
+        }
+
+        service {
+            connect {
+                sidecar_service {
+                    proxy {
+                        dynamic "upstreams" {
+                            iterator = alpha
+                            for_each = local.dgraph_alphas
+
+                            content {
+                                destination_name = "dgraph-alpha-${alpha.value.id}-grpc-public"
+                                local_bind_port = alpha.value.grpc_public_port
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    group "grapl-node-identifier" {
+        count = var.node_identifiers
+
+        network {
+            mode = "bridge"
+        }
+
+        task "node-identifier" {
+            driver = "docker"
+
+            config {
+                image = "${var.container_registry}/node-identifier:${var.tag}"
+                args = ["/node-identifier"]
```

Comment:
This is the container's entrypoint now.

This applies to all the other Rust service containers, as well.